samples/client/mockserver/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM rodolpheche/wiremock as mockserver
+COPY config /home/wiremock

```

Comment:
I think it would be nice to add a 3rd way to add a docker job by specify the docker hub name, or did I miss it?

Reply:
I think we could add another property `dockerSourceImage`, and when present instead of doing a `docker build` doing a `docker pull` then tagging the image with the `dockerImageName` so that the rest of the logic stay the same.
What do you think @sebastienros ? would you accept a PR for that?

Reply:
I agree with the suggestion. Had the same issue myself which required me to create a dumb docker file. `dockerImage` ? Would allow for name and tag. And change how dockerImageName is constructed when not provided to be sure the tag part is not interfering with the syntax of the command line.